### PR TITLE
dialects/linux/machine: remove sysvlegacy function

### PR DESCRIPTION
**Pull request by ffontaine**
This patch has been added in the context of buildroot fifteen years ago: https://git.buildroot.net/buildroot/commit/package/lsof?id=c1efa9b291f0aa1453ea6359cc6638c0e75b2ca7

[Retrieved from: https://git.buildroot.net/buildroot/tree/package/lsof/0002-remove-susvlegacy-funcs.patch]

While at it also fix indentation in 00DIST

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

### CODE REVIEW

File: 00DIST [hunk 1]
```diff
@@ -5295,11 +5295,14 @@ July 14, 2018
...
 
-               Fix broken LSOF_CFLAGS_OVERRIDE.
-               Provided by Fabrice Fontaine in #172.
+		Fix broken LSOF_CFLAGS_OVERRIDE.
+		Provided by Fabrice Fontaine in #172.
 
-               Fix broken AR and RANLIB when cross-compiling.
-               Provided by Fabrice Fontaine in #194.
+		Fix broken AR and RANLIB when cross-compiling.
+		Provided by Fabrice Fontaine in #194.
```
Comment (masatake) [inline 00DIST:5302]: Could you make a separate commit for this "docs: convert spaces to tabs" change?
This change is nothing to do with "dialects/linux/machine: remove sysvlegacy function ".

File: 00DIST [hunk 2]
```diff
@@ -5295,11 +5295,14 @@ July 14, 2018
...
+
+		Remove sysvlegacy function.
```
Comment (masatake) [inline 00DIST:5304]: Could you add `[linux]` as a prefix because this change is for the Linux dialect?
In addition, could you add `[linux]` to the header of the commit log like:
```
[linux]  dialects/linux/machine: remove sysvlegacy function 
```

**Review by masatake — CHANGES_REQUESTED** [review]

**Comment by masatake** [comment]
Thank you for your contribution.

I have one meta-comment. You use `master` branch of YOUR repository.
I recommend you to make a topic branch per pull request instead of committing changes to your `master` branch.
I made similar advice to a contributor at another project: https://github.com/universal-ctags/ctags/pull/2684 .

**Comment by masatake** [comment]
I think [354f135](https://github.com/lsof-org/lsof/pull/195/commits/354f13547aed0a52a83b40499970cd85d0f07d48) and [8f21db8](https://github.com/lsof-org/lsof/pull/195/commits/8f21db898ef92bfa2d8ac3c8743eccf89b5c6311) are not needed in this pull request.  Could you remove the two? `git rebase -i 354f135^` may help you.

**Comment by ffontaine** [comment]
Thanks for your review, it should be ok now.

**Comment by masatake** [comment]
Thank you!
